blockchains/matic/lib/fetch_events.js

```diff
@@ -32,11 +32,12 @@ const decodeFunctions = {
 
 };
 
-async function getPastEvents(web3Wrapper, fromBlock, toBlock) {
+async function getPastEvents(ethClient, web3Wrapper, fromBlock, toBlock) {
   const events = await getRawEvents(web3Wrapper, fromBlock, toBlock);
 
-  const decodedEvents = decodeEvents(web3Wrapper, events, new TimestampsCache(), decodeFunctions);
```

Comment:
I have previously missed to update the construction of TimestampsCache() in this context.